pkg/kn/commands/service/service_update_test.go

```diff
@@ -88,6 +88,26 @@ func fakeServiceUpdate(original *v1alpha1.Service, args []string, sync bool) (
 	return
 }
 
+func TestServcieUpdateNoFlags(t *testing.T) {
+	orig := newEmptyService()
+
+	action, _, _, err := fakeServiceUpdate(orig, []string{
+		"service", "update", "foo"}, false)
+
+	if action != nil {
+		t.Errorf("Unexpected action if no flag(s) set")
+	}
+
+	if err != nil {
```

Comment:
I think you can reduce a if-else depth by swapping the if-else block and using t.Fatal.

Reply:
@toVersus I addressed it, thanks!